### PR TITLE
OMNI-1259 Expose product image resource to Client API

### PR DESCRIPTION
**Pull request by bc-ruth**
Please review @bc-bfenton @lord2800 @ICarpenter @aleachjr

### CODE REVIEW

File: test/Unit/Api/ClientTest.php [hunk 1]
```diff
@@ -400,9 +400,7 @@ public function testGettingProductImagesReturnsCollectionOfProductImages()
 
         $collection = Client::getProductImages(1);
         $this->assertInternalType('array', $collection);
-        foreach ($collection as $resource) {
-            $this->assertInstanceOf('Bigcommerce\\Api\\Resources\\ProductImage', $resource);
-        }
+        $this->assertContainsOnlyInstancesOf('Bigcommerce\\Api\\Resources\\ProductImage', $collection);
```
Comment (lord2800) [inline test/Unit/Api/ClientTest.php]: 👏 Nice find!

**Comment by lord2800** [comment]
It looks like you made Travis quite unhappy, but 👍 for the code.

File: test/Unit/Api/ClientTest.php (hunk 1 above)
Comment (bc-ruth) [inline test/Unit/Api/ClientTest.php]: thanks (: but 
👏 belongs to @bc-bfenton

**Comment by bc-bfenton** [comment]
Tests pass locally just fine. Hmm...

**Comment by lord2800** [comment]
Yeah, it's quite confusing. I'm not entirely sure what to make of it--and all the tests fail for the same reason, too.

**Comment by bc-bfenton** [comment]
👍

**Pull request merged**